flutter_local_notifications_windows/pubspec.yaml

```diff
@@ -0,0 +1,24 @@
+name: flutter_local_notifications_windows
+description: "A new Flutter FFI plugin project."
+version: 1.0.0
+
+environment:
+  sdk: ">=3.1.0 <4.0.0"
+
+dependencies:
+  ffi: ^2.1.2
+  flutter_local_notifications_platform_interface: ^7.2.0
+  timezone: ^0.9.4
+  xml: ^6.5.0
+
+dev_dependencies:
+  ffigen: ^13.0.0
+  very_good_analysis: ^6.0.0
```

Comment:
Could the use of this be removed for the moment? I recall you did raise a comment around linter rules used but for now would want to ensure there's consistency and can revisit the rules used later. If I remember correctly, if the Windows package has its `analysis_options.yaml` file removed, it should pull the one from the root of the repo. This may have required melos to bootstrapped

Reply:
On a related note, I recall you mentioned another comment on Dart line length. I get where you're coming from and at work I work in a team that has chosen a larger value.  However, the default line length is 80. There's also a linter for it and is also being followed in a lot of repos to my knowledge e.g. the first-party packages. Looking to follow the defaults here for consistency

Reply:
Commit 9146d65 removes the custom lints and conforms to the 80 character line rule, as well as all the other lints. While I agree with conformity with the rest of the monorepo, I would strongly suggest reconsidering which lints the monorepo uses. Particularly, `cascade_invocations` forced a lot of imperative-style code like: 
```dart
builder.element(...);
builder.element(...);

plugin.show(...);
plugin.testDetails(...);
```

To be formatted as 
```dart
builder
  ..element(
    // ...
  )
  ..element(
    // ...
  );

plugin
  ..show(
    // ...
  )
  ..show( 
    // ...
  );
```

And of course, with a long name like `FlutterLocalNotifications` and prefixes/suffixes like `Windows` everywhere, the 80-character line length gets violated at almost every line. Even function signatures _before_ the parameters have to be split: 

```dart
  @override
  Future<List<PendingNotificationRequest>>
    pendingNotificationRequests() async =>
      using((Arena arena) {
```

This is made worse by `always_specify_types` requiring very long type names be used even when their constructors are in the same line. 

Again, I already committed the fixes for this PR, this is just a long-term recommendation advice for the rest of the repo. I do generally agree with following standards, these in particular aren't exactly agreed upon. While `always_specify_types` is part of [the Flutter style guide](https://github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-using-var-and-dynamic), it [goes against Effective Dart](https://dart.dev/effective-dart/design#dont-redundantly-type-annotate-initialized-local-variables), and its opposite rule, [`omit_local_variable_types`](https://dart.dev/tools/linter-rules/omit_local_variable_types) is recommended instead. Character limits come from `dart format`, whose description explicitly recommends _not_ using it if your project needs configuration, and I'd say this package having a long name would justify that.

Reply:
Yep noted, thanks